tests/unit/repositories/ContentReadRepositoryTest.php

```diff
@@ -223,5 +223,148 @@ public function canPaginateResults()
         $this->assertEquals('B title', $result->first()->translations->first()->title);
         $this->assertEquals('A title', $result->last()->translations->first()->title);
     }
+
+    /** @test */
+    public function getManyPublishedShouldHideTranslationsForInactiveRoutes()
+    {
+        $this->tester->haveContents([
+            [
+                'published_at' => Carbon::now()->subDays(4),
+                'translations' => [
+                    [
+                        'language_code' => 'en',
+                        'title'         => 'Inactive translation title',
+                        'is_active'     => false // under the hood, we're creating inactive route too
+                    ],
+                    [
+                        'language_code' => 'en',
+                        'title'         => 'Active translation title',
+                        'is_active'     => true
+                    ],
+                    [
+                        'language_code' => 'pl',
+                        'title'         => 'Example active english translation four days ago',
+                        'is_active'     => true
+                    ],
+                ]
+            ]
+        ]);
+
+        $result = $this->repository->getManyPublished(new QueryBuilder());
+
+        $this->assertEquals(1, $result->count());
+        $this->assertEquals(1, $result->total());
+
+        $content = $result->first();
+
+        $this->assertEmpty($content->translations->firstWhere('language_code', 'en'));
+        $this->assertNotEmpty($content->translations->firstWhere('language_code', 'pl'));
+        $this->assertCount(1, $content->routes);
+        $this->assertNotEmpty($content->routes->firstWhere('language_code', 'pl'));
+    }
+
+    /** @test */
+    public function getManyPublishedShouldReturnContentWitchHavePublishedAtDateInThePastAndDifferentThanNull()
+    {
+        $this->tester->haveContents([
+            [
+                'published_at' => Carbon::now()->subDays(1),
```

Comment:
Why are you copying all those contents to see only if it works for null? I should be just a simple test that checks yet another case. We don't need that duplication